YLTableViewTests/YLTableViewBaseTestCase.h

```diff
@@ -0,0 +1,20 @@
+//
+//  YLTableViewBaseTestCase.h
+//  YLTableView
+//
+//  Created by Ushhud Khalid on 12/9/15.
+//  Copyright © 2015 Yelp. All rights reserved.
+//
+
+#import <XCTest/XCTestCase.h>
+
+//! New tests should subclass from this class as it contains contains all common test methods/properties.
+@interface YLTableViewBaseTestCase : XCTestCase
```

Comment:
I'd recommend against having a base test case class here. A base test case makes sense in our codebase and the super warnings are pretty convenient, but for a small open-source project that we want people to be able to pick up quickly and contribute to I feel like this just might add confusion. Especially since we only have one test case now, for now I'd say YAGNI (you ain't gonna need it)!